etcd-client/src/client.rs

```diff
@@ -142,7 +142,7 @@ impl Client {
                 auth_client: Auth::new(AuthClient::new(channel.clone())),
                 kv_client: Kv::new(
                     KvClient::new(channel.clone()),
-                    &etcd_watch_client,
+                    etcd_watch_client.clone(),
```

Comment:
Why clone here?

Reply:
Because Lazy::new() init requires static lifetime, so need to clone and give ownership.